config/mailjet.php

```diff
@@ -14,10 +14,10 @@
         'secret' => env('MAILJET_API_SECRET'),
     ],
     'expiration_timers' => [ // Timer is always in months
-        IntegrationType::EntryApi->value => env('MAILJET_EXPIRATION_TIMER_ENTRY_API', 6),
-        IntegrationType::SearchApi->value => env('MAILJET_EXPIRATION_TIMER_SEARCH_API', 6),
-        IntegrationType::UiTPAS->value => env('MAILJET_EXPIRATION_TIMER_UITPAS', 6),
-        IntegrationType::Widgets->value => env('MAILJET_EXPIRATION_TIMER_WIDGETS', 3),
+        IntegrationType::EntryApi->value => (int)env('MAILJET_EXPIRATION_TIMER_ENTRY_API', 6),
```

Comment:
I'm not sure if this is the best place to fix this, Laravel has built-in support for this https://laravel.com/docs/11.x/configuration#accessing-configuration-values

It is advised to use `Config::integer`

Reply:
:-O O, that is very good to know, I will change it!

Reply:
Oké, now I look into it, while this is good knowledge, I am afraid I cannot use it here.
I don't directly access each variable with Config::get(), I pass the entire array.

This gives me a lot more flexibility for in the future if for example we have to not sent mails for one specific type, or quickly add a type.
I would like to keep it this way.

Alternatively I can do a cast to int in app/Console/Commands/SearchExpiredIntegrations.php if you would prefer this.

Reply:
Indeed valid remark! It's fine to just keep the `int` conversion on the `env` function